File: ruby/7_kyu/cw_finddivisors.rb
def divisors(n)
  divisors = []
  for num in (2...n)
    divisors << num if n % num == 0
  end
  divisors.empty? ? "#{n} is prime" : divisors
end

# https://www.codewars.com/kata/544aed4c4a30184e960010f4 

File: javascript/7_kyu/cw_finddivisors.js
function getDivisorsCnt(n){
  let count = 0;
  for (i = 0; i <= n; i++) {
     if (n % i == 0) {
       count++;
     }
  }
  return count;
}

// https://www.codewars.com/kata/find-divisors-of-a-number/javascript